cmd/commands/init.go

```diff
@@ -53,4 +57,5 @@ The index-url is mandatory. Ex "cpackget init --pack-root path/to/mypackroot htt
 func init() {
 	InitCmd.Flags().BoolVarP(&initCmdFlags.downloadPdscFiles, "all-pdsc-files", "a", false, "downloads all the latest .pdsc files from the public index")
 	InitCmd.Flags().BoolVarP(&initCmdFlags.encodedProgress, "encoded-progress", "E", false, "Reports encoded progress for files and download when used by other tools")
+	InitCmd.Flags().BoolVar(&initCmdFlags.skipTouch, "skip-touch", false, "do not touch pack.idx")
```

Comment:
can you confirm if this needs only long options `--skip-touch` and no short option?

Reply:
@bgn42 invented the option which is needed for PackInstaller. We should not add everything as a short option just to have it, we should keep free the single letters for when we really need them.